docs/source/simulated_populations/index.rst

```diff
@@ -65,6 +65,12 @@ The following table provides the SHA-256 checksum for the larger-scale simulated
   * - Location
     - File
     - SHA-256 checksum
+  * - US
+    - pseudopeople_simulated_population_usa_2_0_0.zip
+    - 0025978196c2a84c1df502e857bec35a84c25092fbfb6b143c0b8ff30dea5eed
+  * - Rhode Island
+    - pseudopeople_simulated_population_ri_2_0_0.zip
+    - bfec148c947096b44201a7961a1b38f63961cd820578f10a479f623d8d79f0d1
   * - US
     - pseudopeople_simulated_population_usa_1_0_1.zip
```

Comment:
I think the original version of the data was 1.0.0? But maybe it doesn't matter because we never published this page? Or do we want the filenames that show up here to match what people actually have? In that case, it was something like `pseudopeople_input_data_usa_1_0_0.zip`, but then we'd be using our old terminology instead of our new terminology.

Reply:
I agree. Im not sure when it got changed to 1_0_1 but I've changed it to match what is on Azure.

Reply:
We changed the version to 1_0_1 as a placeholder when we updated the name from input_data to simulated_population.